src/components/cc-icon/cc-icon.js

```diff
@@ -70,8 +77,11 @@ export class CcIcon extends LitElement {
 
   static get styles () {
     return [
+      skeletonStyles,
       css`
         :host {
+          --cc-skeleton-color: #bbb;
```

Comment:
We're trying to apply this pattern and maybe it should be documented somewhere :+1: 

1. private CSS variable and only used in a component => no `cc-` prefix
2. public CSS variable specific to the component and which can be configured by the user => `cc-foobar-something` prefix `foobar` being the component name
3. public CSS variable global to multiple components => `cc-something` prefix, those mostly come from the theme

We still don't have a variable and proper system for skeleton :-(

I think we should either document the skeleton as a (2) variable or go private (1) and even inline `#bbb` in the only place it's used for now while we wait for a better skeleton system.